### ISSUE TITLE
Inconsistent HTML ~strikethrough~ formatting

### ISSUE THREAD

**Issue by cbrunnkvist** [stryker-mutator/stryker4s#496]
### Summary

I believe the original code should always be crossed-out when any of its mutants are "opened" (i.e. clicked) in the HTML reports. What seems to be happening instead is that only _the last one of them_ actually triggers the `text-decoration: line-through` over its original code.

#### Example A (only mutant "3" uses strikethrough):

<img width="409" alt="Screen Shot 2020-06-08 at 13 28 57" src="https://user-images.githubusercontent.com/34006/84000210-75801580-a98e-11ea-90ac-c0ad4e563aa2.png">
<img width="410" alt="Screen Shot 2020-06-08 at 13 29 05" src="https://user-images.githubusercontent.com/34006/84000208-74e77f00-a98e-11ea-9d96-0b2a6f24fc89.png">

#### Example B (only mutant "8" uses strikethrough):

<img width="476" alt="Screen Shot 2020-06-08 at 13 29 31" src="https://user-images.githubusercontent.com/34006/84000206-73b65200-a98e-11ea-91f3-04c4562c0d91.png">
<img width="456" alt="Screen Shot 2020-06-08 at 13 29 38" src="https://user-images.githubusercontent.com/34006/84000205-731dbb80-a98e-11ea-8221-0222fb10542b.png">
<img width="469" alt="Screen Shot 2020-06-08 at 13 29 45" src="https://user-images.githubusercontent.com/34006/84000196-6ef19e00-a98e-11ea-992b-72c5649b0427.png">

### Stryker4s config

_(not relevant / defaults)_

### Stryker4s environment

```
stryker4s 0.8.0
```

**Comment by hugo-vrijswijk** [comment]
What browser are you opening the report on? I am unable to reproduce this on Firefox or on Chromium Edge. For example, opening the killed mutants on line 80 [here](https://dashboard.stryker-mutator.io/reports/github.com/stryker-mutator/stryker4s/master#maven-plugin/runner/MavenMutantRunner.scala)

**Comment by cbrunnkvist** [comment]
Good point! I should have thought about browser version: I'm seeing it fail in **Safari 13.1** but as expected perhaps, it seems to be working fine in **Chrome 83.0.4103.61**.

(btw, maybe this should have been reported to `mutation-testing-elements`, but I guess it's the same people looking over there 😉)

**Comment by cbrunnkvist** [comment]
<img width="963" alt="Screen Shot 2020-06-08 at 16 30 28" src="https://user-images.githubusercontent.com/34006/84015027-753f4480-a9a5-11ea-8bc0-46050ae585d7.png">
<img width="963" alt="Screen Shot 2020-06-08 at 16 30 33" src="https://user-images.githubusercontent.com/34006/84015041-7b352580-a9a5-11ea-9800-11f072db4ba6.png">
<img width="963" alt="Screen Shot 2020-06-08 at 16 30 38" src="https://user-images.githubusercontent.com/34006/84015046-7bcdbc00-a9a5-11ea-863a-a8856eb7f391.png">
<img width="963" alt="Screen Shot 2020-06-08 at 16 30 41" src="https://user-images.githubusercontent.com/34006/84015048-7c665280-a9a5-11ea-9a0e-aa76cbfab606.png">
<img width="963" alt="Screen Shot 2020-06-08 at 16 30 48" src="https://user-images.githubusercontent.com/34006/84015051-7d977f80-a9a5-11ea-9d59-87adbe403347.png">
